huntsman/tests/test_huntsman.py

```diff
@@ -276,8 +274,6 @@ def test_darks_collection_simulator(pocs, tmpdir):
     exptimes_list = list()
     for target in pocs.observatory.scheduler.fields_list:
         exptime = target['exptime']
-        if not isinstance(exptime, u.Quantity):
-            exptime *= u.second
         if exptime not in exptimes_list:
             exptimes_list.append(exptime)
```

Comment:
I think you need to actually check whether the FITS files are actually created at the end of this test. You can use something like 
`assert(os.path.isfile(filename))`, or you could count the number of FITS files in the relevant directories etc.

Reply:
I'm mainly asserting two things now: 1) To check whether all of the created files are fits, and 2) verify that the number of created fits files equals the theoretical (expected) number of darks.